media/css/stories/includes/lib.scss

```diff
@@ -0,0 +1,202 @@
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Black.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Black.woff') format('woff');
+    font-weight: 900;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-It.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-It.woff') format('woff');
+    font-weight: normal;
+    font-style: italic;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-BoldIt.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-BoldIt.woff') format('woff');
+    font-weight: bold;
+    font-style: italic;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-ExtraLightIt.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-ExtraLightIt.woff') format('woff');
+    font-weight: 200;
+    font-style: italic;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-ExtraLight.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-ExtraLight.woff') format('woff');
+    font-weight: 200;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Light.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Light.woff') format('woff');
+    font-weight: 300;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-LightIt.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-LightIt.woff') format('woff');
+    font-weight: 300;
+    font-style: italic;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-SemiBold.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-SemiBold.woff') format('woff');
+    font-weight: 600;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
+    src:
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Regular.woff2') format('woff2'),
+        url('/media/fonts/stories/source-serif/SourceSerifPro-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Source Serif Pro';
```

Comment:
This is a lot of fonts. Are we really using all of these weights and styles? We should reduce it to just regular and bold. Maybe a regular italic and bold italic if they're actually used anywhere but I haven't noticed any italics.